test/loaders.spec.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const assume = require('assume');
+const fs = require('fs').promises;
+const loaders = require('../src/lib/loaders');
+const nock = require('nock');
+const path = require('path');
+const sinon = require('sinon');
+
+
+describe('Loaders', function () {
+
+  before(function () {
+    sinon.stub(console, 'log');
+  });
+
+  after(function () {
+    sinon.restore();
+  });
```

Comment:
IMHO make these be `beforeEach` and `afterEach` that way you're not leaking state between tests via sinon.

before/after - before/after the scope they exist in (in this case before & after everything within the `describe`, i.e. these will run once).
beforeEach/afterEach - before/after each test in the scope (i.e. these will run once per test within this `describe`)

Reply:
Hmm yeah probably a good call. My thought here was mostly to just stop console logs from happening and since I am not actually checking these logs I only wanted just a single call as opposed to many. But I could see wanting to check the log values later on.